admin/schema.php

```diff
@@ -750,5 +750,19 @@ function installer_db_now() {
 $g_upgrade[198] = array( 'AlterColumnSQL', array( db_get_table( 'user' ), "password C(64) $t_notnull DEFAULT \" '' \"" ) );
 $g_upgrade[199] = array( 'AlterColumnSQL', array( db_get_table( 'user' ), "email C(255) $t_notnull DEFAULT \" '' \"" ) );
 
-# Release marker: 1.3.0-beta.1
+# Release marker: 1.3.0-beta.3
+
+$g_upgrade[200] = array(
+		'CreateTableSQL',array( db_get_table( 'api_token' ), '
+			  id           I UNSIGNED NOTNULL PRIMARY AUTOINCREMENT,
+			  user_id      I DEFAULT \'0\',
+			  name         C(128) NOTNULL,
+			  hash         C(128) NOTNULL,
+			  date_created I UNSIGNED NOTNULL DEFAULT \'0\',
+			  date_used    I UNSIGNED NOTNULL DEFAULT \'0\''
+		) );
```

Comment:
> We normally set DEFAULT value of " \'\' " for NOTNULL strings I believe

These are not strings, they are integers (dates); other dates in the schema are defaulted to `'1'`, not `''`. I'm not really sure what Oracle or MSSQL have to do with this though. In any case, for consistency, we should probably change the default to 1.

We should also change the *user_id* column to `I UNSIGNED` too, to match user.id.